apps/condo/pages/test.tsx

```diff
@@ -0,0 +1,147 @@
+import React, { useState } from 'react'
+const { gql } = require('graphql-tag')
+import { useQuery } from '@apollo/client'
+import { Spin, Button } from 'antd'
+import { PageContent, PageWrapper } from '@condo/domains/common/components/containers/BaseLayout'
+
+/*
+    Example usage of `fetchMore` feature of Apollo Client
+    Create a ticket and update it more than 2 times to have
+    TicketChange created for it.
+    Then open this page and test loading using `fetchMore`
+
+    If you see following error:
+    > TypeError: Cannot read property 'fetchMore' of undefined
+    It is a result of a Hot Module Reloading.
+    Refresh browser page manually.
+ */
+
+const TestPage = () => {
+    console.log('rendering TestPage')
+    return (
+        <PageWrapper>
+            <PageContent>
+                <Ticket/>
+            </PageContent>
+        </PageWrapper>
+    )
+}
+
+const Ticket = () => {
+    const { loading, data } = useQuery(TICKETS_QUERY)
+
+    if (loading) {
+        return <Spin/>
+    }
+    console.log('rendering Ticket')
+
+    const ticket = data.allTickets[0]
+
+    return (
+        <div>
+            <div>
+                Ticket <strong>{ticket.id}</strong>
+            </div>
+            <TicketChanges ticket={ticket}/>
+        </div>
+    )
+}
+
+const TicketChanges = ({ticket}) => {
```

Comment:
is it woring?

Reply:
Yes, It's working.
It's an example of page pattern, where we have a detail page for something (`Ticket`) and a series of related items (`TicketChange`), that should have pagination, based on `fetchMore`.
This pattern does't works with wrappers from @core, but works when detached from wrappers.